cameraServer.py

```diff
@@ -0,0 +1,285 @@
+#!/usr/bin/env python3
+import json
+import time
+import sys
+import numpy as np
+import cv2
+import math
+
+from cscore import CameraServer, VideoSource
+from networktables import NetworkTables
+    
+
+#Magic Numbers
+lowerGreen = (38, 125, 100) #Our Robot's Camera
+higherGreen = (52, 255, 180)
+sampleLowerGreen = (30, 177, 80) #FRC sample images
+sampleHigherGreen = (150, 255, 255)
+lowerTapeColour = (0, 0, 0) #TODO: Ground Tape filter colours
+upperTapeColour = (360, 255, 255) #TODO: Ground Tape filter colours 
+minContourArea = 10
+angleOffset = 13
+rightAngleSize = -14.5
+leftAngleSize = -75.5
+screenSize = (320, 240)
+
+
+
+#Initialisation
+configFile = "/boot/frc.json"
+
+class CameraConfig: pass
+
+team = None
+server = False
+cameraConfigs = []
+
+def readCameraConfig(config):
+    """Read single camera configuration."""
+    cam = CameraConfig()
+
+    # name
+    try:
+        cam.name = config["name"]
+    except KeyError:
+        parseError("could not read camera name")
+        return False
+
+    # path
+    try:
+        cam.path = config["path"]
+    except KeyError:
+        parseError("camera '" + str(cam.name) + "': could not read path")
+        return False
+
+    cam.config = config
+
+    cameraConfigs.append(cam)
+    return True
+
+def parseError(error_str):
+    """Report parse error."""
+    print("config error in '" + configFile + "': " + error_str, file=sys.stderr)
+
+def readConfig():
+    """Read configuration file."""
+    global team
+    global server
+
+    # parse file
+    try:
+        with open(configFile, "rt") as f:
```

Comment:
By default [open](https://docs.python.org/3/library/functions.html#open) uses `r` and `t` when opening a file
```suggestion
        with open(configFile) as f:
```

Reply:
Again, this is not the place to complain about the example code.